exist-core/src/main/java/org/exist/dom/QName.java

```diff
@@ -137,6 +133,7 @@ public String getStringValue() {
      * @deprecated Use for debugging purpose only,
      * use {@link #getStringValue()} for production
      */
+    @Deprecated
```

Comment:
I am not sure that `@Deprecated` makes sense here from a Java API perspective, as every Java object can have a `toString` method. The `@deprecated` in the comments is more of a suggestion to the developer than anything else.

Reply:
Then I'd suggest to remove the misleading @deprecated from the DocBlock as well.

Reply:
The `@deprecated` in the JavaDoc is not misleading. The `@deprecated` and `@Deprecated` annotations have different purposes, they don't always have to be used together!

Reply:
I reworded the comment in JavaDoc. I think it the point is quite clear that it should not be used.
Why is not so clear on the other hand. It jus calls getStringValue internally anyway.